### PR TITLE
[Core] Removing tabulations on ModelPartIO

### PR DESCRIPTION
**Pull request by loumalouomega**
This is a simple clean up (deactivate whitespaces comparation) on the ModelPartIO

### CODE REVIEW

File: kratos/sources/model_part_io.cpp [hunk 1]
```diff
@@ -25,4743 +25,5056 @@
...
-            (*mpStream) << "\t" << it_node->Id() << "\t" << it_node->X()  << "\t" << it_node->Y() << "\t" << it_node->Z() << std::endl;
-        (*mpStream) << "End Nodes" << std::endl << std::endl;
-    }
+    return true;
+    KRATOS_CATCH("")
+}
 
-    void ModelPartIO::ReadProperties(Properties& rThisProperties)
+/***********************************************************************************/
+/***********************************************************************************/
```
Comment (philbucher) [inline kratos/sources/model_part_io.cpp]: I think those are unnecessary, increase the file size for no reason...

**Review by philbucher — COMMENTED** [review]
fine for me

Since this has the potential for many conflicts I would leave it open for some days such that others can see it

File: kratos/sources/model_part_io.cpp (hunk 1 above)
Comment (loumalouomega) [reply kratos/sources/model_part_io.cpp]: Makes easier to identify when a function starts and finishes. I see it very practical

Comment (pooyan-dadvand) [reply kratos/sources/model_part_io.cpp]: I agree with @philbucher that this separators are really unnecessary and only make the file less readable as there are less code per page and more scrolls

Comment (loumalouomega) [reply kratos/sources/model_part_io.cpp]: Removed

**Review by philbucher — APPROVED** [review]
lets hope it won't create too many conflicts :)

**Comment by loumalouomega** [comment]
Thanks

**Pull request merged**